details.py

```diff
@@ -1,20 +1,36 @@
-def contact_info(contact_name,contact_phone_number,contact_email,contact_dic ):
-    contact_dic[contact_name] = [contact_phone_number,contact_email]
-    return contact_dic
-cd = dict()
+import os
+import json
 
-name1 = input("Contact Name: ")
-phone_number1  = int(input("Contact Phone NUmber: "))
-email1 = input("Contact Email Address: ")
-print(contact_info(name1,phone_number1,email1,cd))
+option = input(
+    "1. Create new contact\n2. Edit Contact\n3. Delete contact\n4. Exit program\nChoose action:  "
+).lower()
+if option == "create new contact":
 
-while True:
-    s = input("Enter 'new' if you want to add a new student,enter 'q 'if you want to finish: ")
-    if s == 'new':
-        name = input("Contact name: ")
-        phone_number = int(input("Contact Phone NUmber: "))
-        email = input("Contact Email Address: ")
-        print(contact_info(name,phone_number,email,cd))
+    def contact_info(
+        contact_name, contact_phone_number, contact_email, contact_dic=dict()
+    ):
```

Comment:
Can you add docstrings to this function and any other function you create so one know what it does easily?

Reply:
Okay, done!